app/helpers/application_helper.rb

```diff
@@ -24,82 +24,98 @@ def bootstrap_class_for(flash_type)
-        HERE
+        FAQ_ANSWER
       },
       {
         question: 'What Are You Actually Teaching?',
-        answer: <<~HERE
-          <p>The Odin Project assumes you’re new to web development, so we start out by going over what exactly a web developer does in our <a href="https://www.theodinproject.com/courses/introduction-to-web-development">Introduction to Web Development</a> course.</p>
-          <p>The <a href="https://www.theodinproject.com/courses/web-development-101">Web Development 101 course</a> will give you a chance to explore several of the languages and technologies you’ll need to know on your journey to becoming a web developer.  Web Development 101 introduces HTML, CSS, Javascript, Ruby, Rails, Git, databases and more. We’ve also created mini-projects that give you a chance to test your new knowledge by building your own applications and websites!</p>
-          <p>The Odin Project takes a back-to-front approach to learning Ruby on Rails.  We start with a <a href="https://www.theodinproject.com/courses/ruby-programming">strong foundation in Ruby</a> and an understanding of databases before migrating into <a href="https://www.theodinproject.com/courses/ruby-on-rails">Ruby on Rails (which is a framework built using Ruby).</p>
-          <p>The Odin Project then takes you deep into <a href="https://www.theodinproject.com/courses/html5-and-css3">HTML and CSS</a>, along with <a href="https://www.theodinproject.com/courses/javascript-and-jquery">Javascript and JQuery</a> to give you all the tools necessary to make websites flow, load quickly, and look great.  Along the way you’ll learn Git and deployment so you can build your own portfolio, collaborate with others, and see your work on the web.</p>
-          <p>Finally, The Odin Project will teach you to work with legacy codebases and open-source projects. We’ll also cover areas specifically intended to help when you’re <a href="https://www.theodinproject.com/courses/getting-hired-as-a-web-developer">looking for employment</a>.</p>
-        HERE
+        answer: <<~FAQ_ANSWER
+          <p>The Odin Project assumes you’re new to web development, so we start out by going over what exactly a web developer does in our <%= link_to "Introduction to Web Development", course_path("introduction-to-web-development") %> course.</p>
+          <p>The <%= link_to "Web Development 101 course", course_path("web-development-101") %> will give you a chance to explore several of the languages and technologies you’ll need to know on your journey to becoming a web developer.  Web Development 101 introduces HTML, CSS, Javascript, Ruby, Rails, Git, databases and more. We’ve also created mini-projects that give you a chance to test your new knowledge by building your own applications and websites!</p>
```

Comment:
I don't think`link_to` with the way we render this onto the view. Did it work when ok when you ran it locally?